lib/js/jslib.js

```diff
@@ -3659,6 +3659,7 @@ var _sin = Math.sin; var sin = LINKS.kify(_sin);
 var _cos = Math.cos; var cos = LINKS.kify(_cos);
 var _log = Math.log; var log = LINKS.kify(_log);
 var _log10 = Math.log10; var log10 = LINKS.kify(_log10);
+var _exp = Math.log10; var exp = LINKS.kify(_exp);
```

Comment:
Uhhh...

```
 var _exp = Math.exp; var exp = LINKS.kify(_exp);
```

??

Reply:
Bug, thanks.

Reply:
Fixed now. It would definitely have been a surprising implementation of `exp`.